### PR TITLE
added ryder

### PR DESCRIPTION
**Pull request by rydersitcawich**
Added Ryder Sitcawich to team page.

### CODE REVIEW

File: src/json/members/rydersit.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Ryder Sitcawich",
+    "pennkey": "rydersit",
+    "major": "Computer Science",
+    "school": "Engineering",
+    "bio": "Hey, I'm Ryder and I'm a Sophomore from Minnesota! In my free time I enjoy playing pickup soccer, cooking beef wellington, and losing my money in poker.",
+    "hometown": "Woodbury, MN",
+    "team": "Penn Courses",
+    "roles": ["Frontend"],
+    "photo": "https://imgur.com/aM2cvav",
```
Comment (joyliu-q) [inline src/json/members/rydersit.json:10]: This is a website url, not an image URL. You need to right click on the photo and select "Copy Image Address".

Comment (joyliu-q) [inline src/json/members/rydersit.json]: This is a website url, not an image URL. You need to right click on the photo and select "Copy Image Address".

**Review by joyliu-q — CHANGES_REQUESTED** [review]

File: src/json/members/rydersit.json (hunk 1 above)
Comment (rydersitcawich) [reply src/json/members/rydersit.json]: Sorry about that. Should be fixed now. Thanks Joy!

**Review by joyliu-q — APPROVED** [review]
LGTM!

**Pull request merged**